react-to-webcomponent.js

```diff
@@ -20,12 +20,22 @@ var define = {
 	}
 }
 
-export default function(ReactComponent, React, ReactDOM) {
+
+/**
+ * Converts a React component into a webcomponent by wrapping it in a Proxy object.
+ * @param {ReactComponent}
+ * @param {React}
+ * @param {ReactDOM}
+ * @param {Object} options - Optional parameters
+ * @param {String?} options.shadow - Use shadow DOM rather than light DOM.
+ */
+export default function(ReactComponent, React, ReactDOM, options= {}) {
```

Comment:
Is `options = {}` valid in every browser that supports Proxy?  If not, we might need to write this out "old school".  I don't think we are currently transpiling this file.

Reply:
Yes.
The only known* difference according to caniuse is Edge 12 and 13 supporting default function parameters and _not_ supporting Proxy.
https://caniuse.com/mdn-javascript_functions_default_parameters
https://caniuse.com/proxy

| * (some browsers are listed in the source docs with unknown feature sets)